fix for failing test:
  The encoding pragma is no longer supported. Check cperl at t/test_non_ascii.t line 10.
  BEGIN failed--compilation aborted at t/test_non_ascii.t line 10.
see https://rt.cpan.org/Public/Bug/Display.html?id=115098
diff -rup XML-DOM-XPath-0.14-_1HaPx-orig/t/test_non_ascii.t XML-DOM-XPath-0.14-_1HaPx/t/test_non_ascii.t
--- XML-DOM-XPath-0.14-_1HaPx-orig/t/test_non_ascii.t	2016-07-14 22:05:39.000000000 -0700
+++ XML-DOM-XPath-0.14-_1HaPx/t/test_non_ascii.t	2016-07-14 22:05:56.000000000 -0700
@@ -7,7 +7,7 @@ use strict;
 use Test::More tests => 10;
 use XML::DOM::XPath;
 
-use encoding 'utf8';
+use utf8;
 
 my $display_warning=0;
 
